<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrievePassword</class>
 <widget class="QWidget" name="RetrievePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QWidget{ background-color: qlineargradient(spread:pad, x1:0.505618, y1:0, x2:0.494, y2:0.806818, stop:0 rgba(255, 171, 195, 255), stop:1 rgba(252, 245, 234, 255)); };

QFrame { background-color: transparent; };</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qss.qrc">:/icon/findPswLabel.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qss.qrc">:/icon/unLabel.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView,QLineEdit{ background-color: #fcf5ea; border: 1px solid #DCDFE6; border-radius: 6px; padding: 5px 10px; font-size: 20px; }
QLineEdit:hover{ border: 1px solid #ff577a; }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>54</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qss.qrc">:/icon/emailLabel.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView,QLineEdit{ background-color: #fcf5ea; border: 1px solid #DCDFE6; border-radius: 6px; padding: 5px 10px; font-size: 20px; }
QLineEdit:hover{ border: 1px solid #ff577a; }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView,QLineEdit{ background-color: #fcf5ea; border: 1px solid #DCDFE6; border-radius: 6px; padding: 5px 10px; font-size: 20px; }
QLineEdit:hover{ border: 1px solid #ff577a; }</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { border-style: none;border: 1px solid #DCDFE6;border-radius: 6px;padding: 5px 10px;font-size: 20px; }
QPushButton:hover { color: #ffffff; font-weight: bold; background-color: #fd839a; }</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { border-style: none;border: 1px solid #DCDFE6;border-radius: 6px;padding: 5px 10px;font-size: 20px; }
QPushButton:hover { color: #ffffff; font-weight: bold; background-color: #fd839a; }</string>
   </property>
   <property name="text">
    <string>返回登录</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qss.qrc"/>
 </resources>
 <connections/>
</ui>
